browser/src/control/Control.ContextMenu.js

```diff
@@ -55,7 +55,7 @@ L.Control.ContextMenu = L.Control.extend({
 			text: ['TableInsertMenu',
 				   'InsertRowsBefore', 'InsertRowsAfter', 'InsertColumnsBefore', 'InsertColumnsAfter',
 				   'TableDeleteMenu', 'SetObjectToBackground', 'SetObjectToForeground',
-				   'DeleteRows', 'DeleteColumns', 'DeleteTable',
+				   'DeleteRows', 'DeleteColumns', 'DeleteTable', 'EditCurrentRegion',
```

Comment:
Once an uno command is allowed in this list, we should check if the dialog behaves correctly in the context of Online. This dialog has a frame called "Link", which allows the section to take content from some other file's section instead of providing own content. I would say this is confusing at best in the context of Online, since the file picker will just show you files in the chroot. The simplest would be to hide the entire Link frame in the Online case. Could you please do that?

That'll be again a core.git change: sw/source/ui/dialog/uiregionsw.cxx:393 has an example on how to hide some widgets in the "web" (Writer/Web, HTML edit mode), you could do something similar:

- check for comphelper::LibreOfficeKit::isActive() to see if this is Online or desktop
- rename frame2 to something like linkframe in the .ui file
- extend the ctor to have a reference to the link frame (probably you can use weld_frame() there)
- hide that entire frame in the LOK case

Could you please do this? Fine to do this as a separate core.git commit. Once that's done, this can also go in. Thanks. :-)

Reply:
Thanks. Let's get to that once https://gerrit.libreoffice.org/c/core/+/161828 is in.